spyder/utils/encoding.py

```diff
@@ -237,18 +238,24 @@ def write(text, filename, encoding='utf-8', mode='wb'):
     else:
         # Based in the solution at untitaker/python-atomicwrites#42.
         # Needed to fix file permissions overwritting.
-        # See spyder-ide/spyder#9381.
+        # See untitaker/python-atomicwrites#17

```

Comment:
@steff456, you made the change I suggested in the wrong place. Please leave here:

    # See spyder-ide/spyder#9381.